content/en/cosign/overview.md

```diff
@@ -15,98 +15,84 @@ features:
+The basic signing format for a blob is as follows:
 
-### 1. Generate a keypair
 
-```shell
-$ cosign generate-key-pair
-Enter password for private key:
-Enter again:
-Private key written to cosign.key
-Public key written to cosign.pub
 ```
-
-### 2. Sign a container and store the signature in the registry
-
-```shell
-$ cosign sign --key cosign.key user/demo
-Enter password for private key:
-Pushing signature to: index.docker.io/user/demo:sha256-87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8.sig
+$ cosign sign-blob <file> --bundle txtbundle.bundle
 ```
 
-The `cosign` command above prompts the user to enter the password for the private key.
-The user can manually enter the password, or set an environment variable with `COSIGN_PASSWORD` to use a password automatically.
-
-### 3. Verify a container against a public key
+The bundle contains signing metadata, including the signature and certificate.  
 
-This command returns `0` if *at least one* `cosign` formatted signature for the image is found
-matching the public key. Review the other sections of this site for information and caveats on other signature formats.
+The Cosign command requests a certificate from our Certificate Authority Fulcio. Fulcio checks your identity by using an authentication protocol (OpenID Connect) to look at your email address. If your identity is correct, Fulcio grants a short-lived, time-stamped certificate. The certificate is bound to the public key to attest to your identity.  This activity is logged using Rekor.
+ 
+Note that you don’t need to use a key to sign.  Currently, you can authenticate with Google, GitHub, or Microsoft. For more information, read [Keyless Signatures](https://docs.sigstore.dev/cosign/keyless/).
 
-Any valid payloads are printed to `stdout`, in JSON format.
-Note that these signed payloads include the digest of the container image, which is how we can be
-sure these "detached" signatures cover the correct image.
+Cosign has additional options and features.   For more information enter the command:
 
-```shell
-$ cosign verify --key cosign.pub dlorenc/demo
-The following checks were performed on these signatures:
-  - The cosign claims were validated
-  - The signatures were verified against the specified public key
-{"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"sha256:87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8"},"Type":"cosign container image signature"},"Optional":null}
+```
+cosign sign-blob --help
 ```
 
-## Kubernetes Integrations
+### Verifying a signed blob
 
-Cosign comes with a few built-in Kubernetes integrations: `Secret` generation, and a [policy webhook](/policy-controller/overview/) `policy-controller`.
-In addition to the `policy-controller`, Cosign is also compatible with and supported by other policy engines such as:
+To verify a signed blob, you need to provide three pieces of information:
+* The certificate, found in the bundle
```

Comment:
Will a new user (assuming someone who knows nothing about Sigstore but lands here to verify something) know what bundle you're referring to? Maybe I'm overthinking this, but is there potential for confusion?

Reply:
+1. I like describing the pieces of information you need, then you can say something like "You may be provided a bundle that contains the certificate and signature".

Reply:
Rewrote section to indicate that the certificate and signature are not always in the bundle, but you may be provided with a bundle that has them.